gkeys/gkeys/actions.py

```diff
@@ -514,6 +547,31 @@ def speccheck(self, args):
                 'SPEC Approved..........: %d' % len(set(failed['spec-approved'])),
             ])
 
+    def _find_template_path(self):
+        config_dir = self.config.get_key('userconfigdir')
+        template_dir = os.path.join(config_dir + '/templates')
+        self.logger.debug("User template path is:" + template_dir)
+        template_path = '/usr/share/gkeys/templates/email_template'
+        if os.path.exists(template_dir + '/email_template'):
+            template_path = template_dir + '/email_template'
+        return template_path
+
+    def _generate_template(self, template_path, keyprints, specprint):
+        with open(template_path, 'r') as file_contents:
+            content = file_contents.read()
+        message_template = Template(content)
+        subkey_msg = ''
+        message = message_template.substitute(key_print=keyprints, spec_print=specprint)
+        return message
+
+    def _is_expiring(self, results, days_limit):
+        is_expiring = False
+        for key in results:
+            for specs in results[key]:
+                if specs.days > days_limit*(-1) and specs.days < days_limit:
+                    is_expiring = True
+                    break
+        return is_expiring
```

Comment:
These 2 utility functions should be re-located to a different file, not in actions.py.  Neither function makes use of or needs self.  So if kept in the class it should get at least an @staticmethod decorator.